Prepare Core and Identity releases (#2527)
Azure/azure-sdk-for-cpp: sdk/core/azure-core/CHANGELOG.md

# Release History

## 1.1.0 (2021-07-01)

> ### Bug Fixes

Reviewer: ```suggestion
### Bugs Fixed
```

- Fixed a memory leak issue in `Base64Encode()`. (A community contribution, courtesy of _[jorgen](https://github.com/jorgen)_)

> ### Other Changes and Improvements

Reviewer: ```suggestion
### Other Changes
```

The headings we want to use are:
`Features Added`, `Breaking Changes`, `Bugs Fixed`, `Other Changes`
https://github.com/Azure/azure-sdk/blob/26e40bb0a9444eb7c7e57d0d1be4a6cf04f3fcbb/docs/policies/releases.md#change-logs

@antkmsft can you make the changes for the new CL entries for all three packages?

> - Added internal support for Azure Key Vault client library.

Reviewer: nit:
```suggestion
- Made internal-only changes to support the Azure Key Vault client library.
```

# Release History

## 1.1.0 (2021-07-01)

### New Features

> - Added internal support for Azure Key Vault client library.

Reviewer 2: Put this in other changes.  Its not customer facing or affecting so calling it out in the release notes does NOT make sense
